backend/include/RadioHelper.hpp

```diff
@@ -80,5 +85,65 @@ class RadioHelper {
 
         return true;
     }
+
+    static void setAllRadioVolumes()
+    {
+
+        auto states = mClient->getRadioState();
+        for (const auto& state : states) {
+            setRadioVolume(state.first);
+        }
+    }
+
+    static void setRadioVolume(
+        const unsigned int frequency, const std::optional<float> volume = std::nullopt)
+    {
+
+        float stationVolume = 100;
+        if (volume.has_value()) {
+            stationVolume = volume.value();
```

Comment:
Shouldn't this value get clamped before the rest of the math code runs?

Reply:
what do you mean?